test/langtools/tools/javac/reflect/CodeReflectionTester.java

```diff
@@ -37,6 +37,8 @@
 
 public class CodeReflectionTester {
 
+    static int nErrors = 0;
```

Comment:
I've tweaked this test so that _all_ errors can be reported at once. This should make it easier to regenerate some of the expected test output when it breaks due to API/compiler changes.